udata/features/territories/views.py

```diff
@@ -72,7 +72,7 @@ def compute_territory_dataset(territory, dataset, resource_id):
                      attachment_filename=attachment_filename)
 
 
-@blueprint.route('/territory/regions/', endpoint='home')
+@blueprint.route('/territoires/', endpoint='home')
```

Comment:
translatable URLs (http://librelist.com/browser/flask/2012/3/19/translatable-url-patterns/ can be interesting) ?